common-npm-packages/packaging-common/universal/ArtifactToolUtilities.ts

```diff
@@ -60,7 +60,7 @@ export async function getArtifactToolFromService(serviceUri: string, accessToken
         }
     }
```

Comment:
Do we have to any this condition, IIRC we were getting IA32 for arm64 arch as well?